### ISSUE TITLE
Cannot Integrate in Rails App

### ISSUE THREAD

**Issue by jalena-penaligon** [Kentico/kontent-delivery-sdk-ruby#31]
### Brief bug description

I am trying to follow this tutorial: https://kontent.ai/blog/creating-a-kentico-cloud-ruby-on-rails-application to integrate Kentico Kontent into an existing Rails v6.0 application. 

I have installed both of these gems:
`gem 'delivery-sdk-ruby'`
`gem 'render_async'`

I added the snippet below to my ApplicationController and replaced <your-project-id> with my Delivery API Project ID. 

  `PROJECT_ID = '<your-project-id>'.freeze`
   `@@delivery_client = KenticoCloud::Delivery::DeliveryClient.new project_id: PROJECT_ID`

At this point, I am receiving the following error message: 'NameError: uninitialized constant ApplicationController::KenticoCloud'.

I tried to dig into the dancing_goat example from this repo and noticed a few discrepancies:
- It's using gem 'kontent-delivery-sdk-ruby', which shows the following error message when attempting to install: 'Could not find gem 'kontent-delivery-sdk-ruby' in any of the gem sources listed in your Gemfile.'
- The delivery_client is being assigned with Kentico::Kontent::Delivery::DeliveryClient.new instead of KenticoCloud::Delivery::DeliveryClient.new. Which is correct?

What am I missing to get this integration up and running?

**Comment by kentico-ericd** [comment]
@jalena-penaligon Thank you for the bug report! The Dancing Goat demo project (and the related article) do need some updating. I've made a few changes and I believe it should be working now.. it was using a very old version of the SDK and also missing some dependencies added in later versions.

Let me know if it works for you!

**Comment by jalena-penaligon** [comment]
@kentico-ericd Thanks so much! I looked through your updates, and updated my project to include gem 'delivery-sdk-ruby' and to require 'kontent-delivery-sdk-ruby' in my application_controller.rb. 

At this point I'm receiving this error:
`Puma caught this error: cannot load such file -- kontent-delivery-sdk-ruby (LoadError)`

Is there anything additional I need to include or require?

**Comment by kentico-ericd** [comment]
@jalena-penaligon Did you also update the version that the dancing_goat project uses? It was using a very old version before the change to Kontent. That should be the last bit to update

**Comment by jalena-penaligon** [comment]
@kentico-ericd Thanks for the follow up Eric, I really appreciate it! I updated the version, and have the dancing_goat project up and running, but I'm still having issues replicating this setup in an existing Rails app that I would like to use Kontent.ai for. 

In my Gemfile, I have added & bundled with these gems:
`gem 'delivery-sdk-ruby'`
`gem 'render_async'`

I have included `require 'kontent-delivery-sdk-ruby'` at the top of my application_controller.rb file, which is throwing a `Puma caught this error: cannot load such file -- kontent-delivery-sdk-ruby (LoadError)` error. 

Is there an additional step needed to load 'kontent-delivery-sdk-ruby' in ApplicationController?
